pkg/controllers/management/rbac/rbac.go

```diff
@@ -20,10 +20,6 @@ const (
 	OwnerAccess                     = "owner"
 	MemberAccess                    = "member"
 	ReadOnlyAccess                  = "read-only"
-	MultiClusterAppResource         = "multiclusterapps"
-	MultiClusterAppRevisionResource = "multiclusterapprevisions"
-	GlobalDNSResource               = "globaldnses"
-	GlobalDNSProviderResource       = "globaldnsproviders"
```

Comment:
Those two don't seem related to catalog v1. Can you confirm if they are deprecated?